### ISSUE TITLE
Queried meta tags missing content property accessor

### ISSUE THREAD

**Issue by bcomnes** [WebReflection/linkedom#191]
When I run 

```js
const { document } = parseHTML(html)
```

And the run the following query on a document with a meta tag with populated content attribute:

```js
document.querySelector("meta[name='description']")?.content
```

The content accessor returned `undefined` even though the queried node as a content attribute with content in it. 

Am I doing something wrong or is this a bug?

**Comment by bcomnes** [comment]
Let me get you a gist.

**Comment by bcomnes** [comment]
The selector works fine, pulls up the node just fine. I just can't access the content property on it. Putting together a gist hold tight.

**Comment by WebReflection** [comment]
I see ... it's possible the meta tag is not understood by the engine, so you should use `getAttribute` instead or bring meta among other tags that are understood and provide accessors

**Comment by WebReflection** [comment]
you can see [meta](https://github.com/WebReflection/linkedom/blob/main/esm/html/meta-element.js) doesn't registry itself like [others](https://github.com/WebReflection/linkedom/blob/main/esm/html/option-element.js) and it doesn't expose any accessor. PR welcome 🙏

**Comment by bcomnes** [comment]
Would it really be as simple as 

```
  get value() { return stringAttribute.get(this, 'value'); }
  set value(value) { stringAttribute.set(this, 'value', value); }
```

**Comment by WebReflection** [comment]
yes, but don't forget to register the tag https://github.com/WebReflection/linkedom/blob/main/how-to-contribute.md#htmlclasses-contributions 👋

**Comment by bcomnes** [comment]
I'll give it a closer read. Thanks for the pointers

**Comment by WebReflection** [comment]
I'll re-pen this so you can commit `Fix #191 - Added <meta> goodness` whenever the PR lands.

**Comment by WebReflection** [comment]
This has been fixed via https://github.com/WebReflection/linkedom/pull/192

**Comment by bcomnes** [comment]
Thank you! I was going to try this weekend but got sidetracked by the downtime on fly.
